### ISSUE TITLE
Multiple reviews in the same project

### ISSUE THREAD

**Issue by psorianom** [asreview/asreview#1728]
## Feature Request

**Is your feature request related to a problem? Please describe.**

Hi, just starting to use asreview. I read in the asreview-insights documentation that in order to plot different simulations, we need to have two files, one for each simulation/model, such as :
```python
with open_state("tests/asreview_files/sim_van_de_schoot_2017_1.asreview") as s1:
    with open_state("tests/asreview_files/"
                    "sim_van_de_schoot_2017_logistic.asreview") as s2:
        plot_recall(ax,
                    [s1, s2],
                    legend_values=["Naive Bayes", "Logistic"],
                    legend_kwargs={'loc': 'lower center'})
```

**Describe the solution you'd like**
I do wonder if it is possible to contain multiple simulations on the same `ASReviewProject` object, and then of course save it to disk.

Sorry if this has been covered elsewhere. Thanks for the library, it looks great 👍

**Comment by Rensvandeschoot** [comment]
Are you perhaps looking for the functionality available in https://github.com/asreview/asreview-makita??

**Comment by J535D165** [comment]
@PeterLombaers what do you think? I think technically, everything is in place to make this work. The only thing that is lacking is a proper integration in the Project API. Shall we work on this for version 2?

**Comment by PeterLombaers** [comment]
Yeah, that would be a nice step. Then you could also reuse feature matrices in case you use the same feature extraction method for multiple simulations. We can have a look!
